__tests__/app.test.js

```diff
@@ -88,7 +93,7 @@ describe('API ROUTES', () => {
 
       royce = playerOne.body;
 
-      donte.userId = user.id;
+      donte.userId = 2;
```

Comment:
this seems off to me. I think you'll want assign a user object not just assign a variable.